include/swift/AST/DiagnosticsDriver.def

```diff
@@ -121,6 +121,9 @@ ERROR(error_conflicting_options, none,
       "conflicting options '%0' and '%1'",
       (StringRef, StringRef))
 
+WARNING(verify_debug_info_requires_debug_option,none,
+        "ignoring '-verify-debug-info'; no debug option was set", ())
```

Comment:
This still looks odd when using -gnone.
What about:
ignoring '-verify-debug-info'; no debug info is being generated